src/main/java/hu/oe/nik/szfmv/automatedcar/sensor/Camera.java

```diff
@@ -0,0 +1,192 @@
+package hu.oe.nik.szfmv.automatedcar.sensor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import hu.oe.nik.szfmv.automatedcar.bus.VirtualFunctionBus;
+import hu.oe.nik.szfmv.automatedcar.bus.packets.CameraPacket;
+import hu.oe.nik.szfmv.automatedcar.systemcomponents.SystemComponent;
+import hu.oe.nik.szfmv.environment.WorldObject;
+import hu.oe.nik.szfmv.environment.worldobjectclasses.RoadSign;
+
+public class Camera extends SystemComponent implements ISensor {
+
+    private static final int VISUAL_RANGE = 80;
+    private static final int HAROMSZAZHATVAN = 80;
+    private static final int METER_PIXEL_RATIO = 50;
+    // 60°
+    private static final double ANGLE_OF_VIEW = 60f;
+    private static double TRIANGULAR_STEM;
+    // camera position
+    private double x;
+    private double y;
+
+    // another 2 point of the triangle
+    private double[] triangleAPoint;
```

Comment:
Ezt lehet érmes lenne defaultba egy triangleba tárolni mert az ha jól felvan paraméterezve kiszámolja a két másik pontot is.

Reply:
felvettem egy triangle-t private field-nek, ahogy mondtad és így egy csomo kódot refactorálni tudtam ...